src/praisonai-agents/praisonaiagents/runtime/tool_output_store.py

```
@@ -0,0 +1,183 @@
+        
+        try:
+            output_path.write_text(output, encoding='utf-8')
+            
+            return {
+                'path': str(output_path),
+                'size': len(output),
+                'tool': tool_name,
+                'call_id': call_id,
+                'timestamp': time.time()
+            }
+        except Exception as e:
```

Ruff (0.15.17)

[warning] Do not catch blind exception: `Exception`

(BLE001)